charts/enterprise/authelia/questions.yaml

```
@@ -346,7 +364,7 @@ questions:
                       description: "The url to the ldap server. Format: <scheme>://<address>[:<port>]"
                       schema:
                         type: string
-                        default: "ldap://openldap.default.svc.cluster.local"
+                        default: "ldap://lldap-ldap.ix-lldap.svc.cluster.local:3890"
```

```suggestion
                        default: "ldap://openldap.default.svc.cluster.local"
```

VETO'ed, the original was a more clear example, the cluster-fucky 3x lldap is making it less clear, note more